bazel/.gcloudignore

```diff
@@ -0,0 +1,3 @@
+cloudbuild.yaml
```

Comment:
why is that?

Reply:
`gcloud` reads the `cloudbuild.yaml` and converts it into the build request; there is no need to submit it to the `/workspace` for the build.